website/templates/emails/new_public_project.html.mako

```diff
@@ -8,7 +8,7 @@
         Congratulations on making a public project on the Open Science Framework (OSF)! Now that your project “${project_title}" is public, you’ll be able to take advantage of more OSF features:
         <ul>
             <li><a href="${osf_url}${nid}/files/">The number of downloads of your files will be automatically logged for you</a></li>
-            <li><a href="${osf_url}${nid}/settings/">You can enable other OSF users to comment on your work</a></li>
+            <li><a href="${osf_url}${nid}/settings/">You can enable other OSF users to comment on your work or update your own comment notification settings</a></li>
```

Comment:
Also, is this to spec? Seems unnecessary (there's a lot that you can do on the settings page that don't need to mentioned in this email).